pypeit/core/skysub.py

```diff
@@ -596,6 +596,8 @@ def local_skysub_extract(sciimg, sciivar, tilts, waveimg, global_sky, rn2_img,
             Show the
         use_2dmodel_mask (bool, optional):
             Use the mask made from profile fitting when extracting?
+        no_local_sky (bool, optional):
+            Do not fit local sky, only object profile and extract optimally
```

Comment:
If True, do not fit the local sky model. Only fit the object profile using the globally sky subtracted image, and then extract optimally. Note that the objimage image will be all zeros if this option is set since the object model is obtained via fitting the local sky.

Reply:
edited